### DISCUSSION TITLE
Layout

### DISCUSSION THREAD

**Discussion by Nexulo** [photoprism/photoprism#1408] [General]
Hello all

I like the idea and also most of the features of Photoprism, but the layout is in my eyes anything but contemporary and mobile-friendly.

For myself, I had developed my own media management on a Raspberry Pi, but then stopped at the face recognition. I am aware of how much work is involved, especially with functions that are executed in the background (indexing, conversion, auto-tagging, reading exif information, etc.). But essential is the structure and layout, where actually simple considerations are very important.

For example:
The most important functions and buttons in the right places. In my opinion, similar to iOS Photos, you should be able to reach the main areas via icons at the bottom of the screen (smartphone) or via the menu bar at the top (desktop) at any time. Always having to open or close a menu is not efficient in my opinion and already much too cumbersome. Also, with the menu at the top or bottom, you always have the same space in the middle area to list the photos.

With all due respect, I think that the layout is very uncharitable.

**Comment by lastzero** [comment]
Why don't you develop the app and we use it for free? I'll let you know when something is missing.

**Reply by Nexulo** [reply]
Of course, I realize that no one likes my words. Nevertheless, the layout is not practical, even if no one wants to hear that. I am certainly not the only one with this opinion.

But if no one has an open ear for it, we can gladly terminate this discussion and leave the layout the way it was done 7 years ago.

**Reply by lastzero** [reply]
Check our roadmap. You think mobile layout is the most important issue? Become a sponsor and vote for it. Or contribute. Windows isn't prefect as well. Billions of users.
